### DISCUSSION TITLE
How to bundle Prisma inside a monorepo as a dependency.

### DISCUSSION THREAD

**Discussion by vorillaz** [prisma/prisma#10030] [Q&A]
Hi there, I am using Prisma inside a factory module. It is actually a builder module that spins off a tailormade Fastify server.
The idea is the I want to build small webservers using a factory as:
```typescript
const server = compose({ name: 'App working' })
  .attach(require('./test'))
  .build()
  .start(port);
```

Using TypeScript I was able to create the appropriate aliases and `esbuild` bundles the output. When it comes to bundling Prisma I am having some really hard times, to appropriately generate the output. 
I am always getting an `Error: ENOENT: no such file or directory app-prisma-fail/dist/schema.prisma ` error.

I guess I should not bindly copy the files around as there are some internal processes and hooks triggered by Prisma.
Any ideas or advice would be really helpful. You may find a reproducable example on this repository: https://github.com/vorillaz/prisma-monorepo-reproduce

Just try to build the app as:
```
npm install
 cd app-working 
 npm run build
 node dist/server.js 
```

And the prisma failing app:
```
 cd app-prisma-fail && npm run build && node dist/server.js 
```

**Comment by austincrim** [comment]
Hey @vorillaz, thanks for the detailed question.

I appreciate the reproduction repo, however I'm unable to run the steps you have outlined. After running `npm run build`, I get the following output:
```bash
server.ts:1:20: error: Could not resolve "@demo/build" (mark it as external to exclude it from the bundle)
```

Any guidance on how I could get this running?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
